admin/controllers/restart.js

```diff
@@ -1,14 +1,18 @@
 const restartService = require('../services/restart.service')
+const restartValidator = require('../lib/validator/restart-validator')
+const ValidationError = require('../lib/validation-error')
 
-const getRestartOverview = async (req, res) => {
+const controller = {}
+
+controller.getRestartOverview = async (req, res) => {
```

Comment:
Is this really async?

Reply:
@jon-shipley nope bad paste from me and good catch from you